webcompat/helpers.py

```diff
@@ -100,6 +100,18 @@ def get_user_info():
         session['avatar_url'] = gh_user.get('avatar_url')
 
 
+def version_string(json_section):
```

Comment:
About Dict - I'm pretty new to Python, sorry :)
It's used not only for `user_agent`, but for `os` too.